Tests/StructuredQueriesTests/TriggerDefaultNameTests.swift

```diff
@@ -0,0 +1,20 @@
+import Dependencies
+import Foundation
+import InlineSnapshotTesting
+import StructuredQueries
+import Testing
+import _StructuredQueriesSQLite
+
+extension SnapshotTests {
+  @Suite struct TriggerDefaultNameTests {
+    @Dependency(\.defaultDatabase) var db
+
+    @Test func basics() {
+      let trigger = RemindersList.createTemporaryTrigger(
+        after: .insert { new in
+          Values(1)
+        }
+      )
```

Comment:
did you want to `assertQuery` this trigger?

Reply:
Haha, yep